portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,7 +37,8 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-  private int maxNumComments;
+  private final int defaultMaxNumComments = 7;
```

Comment:
nit - per [style guide](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names), constants should be UPPER_CASE.

Reply:
Ah I didn't know that, I'll fix that.